dev/run-tests.py

```diff
@@ -574,8 +574,7 @@ def main():
                                 or f.endswith("checkstyle.xml")
                                 or f.endswith("checkstyle-suppressions.xml")
                                 for f in changed_files):
-        # run_java_style_checks()
-        pass
+        run_java_style_checks()
```

Comment:
This is really minor, but now you'll be running this when building with maven too. That could be avoided by checking that the build is not using maven; and also changing the maven target from `package` to `verify` so that these checks run.

You can add `[build-maven]` to the PR title to try it out (or locally with `AMPLAB_JENKINS_BUILD_TOOL=maven`).

Reply:
no problem.